cocos/platform/android/java/src/com/cocos/lib/CocosVideoView.java

```diff
@@ -247,14 +245,9 @@ public void stop() {
         if (!(mCurrentState == State.IDLE || mCurrentState == State.INITIALIZED || mCurrentState == State.ERROR || mCurrentState == State.STOPPED)
                 && mMediaPlayer != null) {
             mCurrentState = State.STOPPED;
-            mMediaPlayer.stop();
+            mMediaPlayer.pause();
```

Comment:
Why pause instead of stop?

Reply:
using stop will cause seekTo to be invalid unless the MediaPlayer is reinitialized.

Reply:
Then there is not difference between pause and stop.

Reply:
There is still a distinction between the upper layer, but the bottom layer uses pause and seekTo to fake the stop behavior, and the web platform is also handled in this way

Reply:
`and the web platform is also handled in this way`

Why all platforms need to do like this? Why support `seekto` after it is stopped?

Reply:
> Why all platforms need to do like this? Why support seekto after it is stopped?
Not to unify it, This is to fix the problem that seekTo cannot be used again without reinitializing after stopping

Reply:
Yep, what i mean is that. Is it needed to support seekTo after it is stopped.

Reply:
Example:

video.play();
video.stop();
video.play();
video.seekTo(10); // invalid

video.play();
video.puase();
video.seekTo(10); // success

Reply:
Then i think the issue is not stopped, because after stopped, it is prepared again by `mMediaPlayer.prepare();`. If media player is prepared, then it can be seek to.

Reply:
![image](https://user-images.githubusercontent.com/7564028/111753780-37773100-88d2-11eb-877e-c68e5f04c307.png)

This is the previous code, but it doesn't work

Reply:
You should know why it can not work, not just avoid it.